xbmc/pvr/recordings/PVRRecording.h

```diff
@@ -82,8 +84,23 @@ namespace PVR
     std::string   m_strFanartPath;    /*!< fanart path */
     unsigned      m_iRecordingId;     /*!< id that won't change while xbmc is running */
 
+  private:
```

Comment:
Seams that we agree the having a private op= and copy ctor is fine. Good.

Regarding declaration only vs decl plus impl: Only declaring it has the advantage that friends and even instances of the class itself are prevented by the linker (yes, the linker, not the compiler) from creating copies. Thus, the point is not (only) to declare them private, but to (also)  NOT implement them at all. If you implement them, you have no chance to automatically detect accidentally copies generated by instances of the class itself (and friends ;-).

Regardig public ctor vs friend (seems to be your main concern): I also hate friend classes, but if the factory pattern can applied I'm fine with it. Sure, a friend always has access to everything in the class, but is giving everyone access to the ctor by making it public really better? I'm note quite sure about this...